.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+
+node_js:
+  - "6.2"
+
+before_script:
+  - npm install -g gulp bower
+  - bower install
```

Comment:
(The readme could also say that `npm test` will run the tests too, but I don't think there's much point in that, it just sticks to the `gulp` methods)